panel/backends/xcb/CMakeLists.txt

```diff
@@ -1 +1,21 @@
+set(NAME xcb_backend)
```

Comment:
Can we drop the `_backend` part from the name? Currently, the libraries get named as `libxcb_backend.so`, `libkwin_wayland_backend.so` and so on. 

It would be better if we named the libraries as `libwmbackend_<platform>.so`. For example, `libwmbackend_xcb.so`, `libwmbackend_kwin_wayland.so`, `libwmbackend_wlroots.so`, so on and so forth.

In the future, should we need to develop "backend" for something else, we will not have the need to worry about filenames.

Reply:
Done!